code/modules/projectiles/gun.dm

```diff
@@ -442,16 +442,18 @@
 		return FALSE
 	return TRUE
 
-/obj/item/gun/proc/handle_suicide(mob/living/carbon/human/user, mob/living/carbon/human/target, params)
-	if(!ishuman(user) || !ishuman(target))
+/obj/item/gun/proc/handle_suicide(mob/living/user, mob/living/carbon/human/target, params)
+	if(!ishuman(target))
```

Comment:
Not necessary to check if the user isliving() here? Or that ishuman(user) can be completely removed?

Reply:
Probably not necessary to check if the user is living, dead people aren't known for using guns.

I can probably remove the human check so any mob can be executed.

Reply:
Did you ask the teams if they want so that any mob can be executed? This is a fix PR focused on cyborgs being able to do it, would be a bit out of scope and is maybe a change that should be voted on.

Reply:
> Did you ask the teams if they want so that any mob can be executed? This is a fix PR focused on cyborgs being able to do it, would be a bit out of scope and is maybe a change that should be voted on.

True, I'll put back the human check. Executing everything else can be a separate PR.